src/editors/containers/VideoUploadEditor/VideoUploader.jsx

```diff
@@ -4,33 +4,62 @@ import { useIntl } from '@edx/frontend-platform/i18n';
 import {
   Icon, IconButton, Dropzone, InputGroup, FormControl,
 } from '@edx/paragon';
-import { ArrowForward, FileUpload } from '@edx/paragon/icons';
+import { ArrowForward, FileUpload, Close } from '@edx/paragon/icons';
 import { useDispatch } from 'react-redux';
 import { thunkActions } from '../../data/redux';
 import * as hooks from './hooks';
+import * as editorHooks from '../EditorContainer/hooks';
 import messages from './messages';
 
 const URLUploader = () => {
+  const [textInputValue, setTextInputValue] = React.useState('');
+  const onURLUpload = hooks.onVideoUpload();
   const intl = useIntl();
   return (
-    <div className="d-flex flex-column flex-wrap">
-      <div className="justify-content-center align-self-center bg-light rounded-circle p-4">
-        <Icon src={FileUpload} className="text-muted" />
+    <div className="d-flex flex-column">
+      <div style={{ backgroundColor: '#F2F0EF' }} className="justify-content-center align-self-center rounded-circle p-5">
```

Comment:
This corresponds to `light-300` so you can use the corresponding class here.

Reply:
@xitij2000 light-300 is #f0eeed which is different. Is that okay?

Reply:
I think the issue here is that the mockups you'll follow will use the "edX" theme, not the base theme used by Open edX. So you need to look up the colours from the edX theme and use that class. It will look different during dev if you don't use the edX branding package, but that's OK.